sample-data/quotenet-output.csv

```diff
@@ -0,0 +1,43 @@
+﻿TEVA PFD 7,Teva Pharmaceutical Industries Ltd 7 % Conv.Cum.Pfd Shs Mandatory Conv. 15.12.2018,IL0062905489

```

Comment:
It's the simplest way I've found to document the output structure/semantics.
It's what Lior was looking for during the hackathon. Not too long and contains a few "not found"s.

I could make input + output examples much shorter (1-2 results + 1 "not found"). @nirgn975 Would you like me to do that?